commands/src/main/java/net/apartium/cocoabeans/commands/CommandManager.java

```diff
@@ -180,6 +180,16 @@ private boolean invokeException(HandleExceptionVariant handleExceptionVariant, C
     }
 
     private boolean invoke(CommandContext context, Sender sender, RegisteredCommandVariant registeredCommandVariant) {
+        context = new CommandContext(
```

Comment:
umm what is that

Reply:
Multiple variant each have its own command info